docs/manuals/core/config/scenario-config.md

```diff
@@ -23,7 +23,7 @@ scenario_config = Config.configure_scenario("multiply_scenario", [pipeline_confi
 In this example, we create a scenario configuration `ScenarioConfig^` from a pipeline configuration already defined.
 
 
-# Scenario configuration from task configurations
+# From task configs
```

Comment:
You use 'configuration' in the text (which I like very much). Why 'config' here?